src/app/i18n.ts

```diff
@@ -100,5 +89,39 @@ export function initi18n(): Promise<unknown> {
         }
       }
     );
+    if (DIM_LANG_INFOS[lang]?.pluralOveride) {
+      i18next.services.pluralResolver.addRule(lang, i18next.services.pluralResolver.getRule('en'));
+    }
   });
 }
+
+function getLanguagePath(language: string) {
+  const languagePath = {
+    de: de,
+    en: en,
+    es: es,
+    fr: fr,
+    it: it,
+    ja: ja,
+    pl: pl,
+    ru: ru,
+    ko: ko,
+    esMX: esMX,
+    ptBR: ptBR,
+    zhCHT: zhCHT,
+    zhCHS: zhCHS,
+  };
+
+  return languagePath[language] as string;
+}
+
+function validLocales() {
+  return DIM_LANGS.map((lang) => {
```

Comment:
you could do

`lang.replace(/(?<=[a-z])(?=[A-Z])/,'-')`

to add a hyphen to any lowerUPPER transition